### PR TITLE
Insure that the bluetooth combobox has the right item selected

### PR DESCRIPTION
**Pull request by nirvn**
When the bluetooth device chooser combobox's Component.onCompleted is triggered, positioningSettings.positioningDevice isn't yet set to the actual value, resulting in the wrong combobox index being selected. The fix is to rely on the 'raw' settings object at that early stage.

Issue didn't impact positioning functionality per say (i.e. the bluetooth device is still picked), but does result in confusing UI/UX. 

@mbernasocchi , we had spotted this issue together while testing GNSS devices on the last hackfest day.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1d95917aff7a5945aa5cd7170dec34325af017da: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3430-e97109c39eff052bc77e06b7416043e41be1a3ca-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3430-e97109c39eff052bc77e06b7416043e41be1a3ca-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3430-e97109c39eff052bc77e06b7416043e41be1a3ca-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3430-e97109c39eff052bc77e06b7416043e41be1a3ca-x86-android.apk)
</details>

File: src/core/positioning/positioning.cpp [hunk 1]
```diff
@@ -135,6 +135,7 @@ void Positioning::setupDevice()
 
   if ( mActive )
   {
+    qDebug() << "CONNN";
```
Comment (m-kuhn) [inline src/core/positioning/positioning.cpp:138]: 💪

Comment (nirvn) [reply src/core/positioning/positioning.cpp]: @m-kuhn , :) fixed

File: src/qml/qgismobileapp.qml [hunk 2]
```diff
@@ -205,6 +205,13 @@ ApplicationWindow {
       skipAltitudeTransformation: positioningSettings.skipAltitudeCorrection
     }
   }
+  Connections {
+    target: positionSource.device
+
+    function onLastErrorChanged() {
+        displayToast(qsTr('Positioning error') + ': ' + positionSource.device.lastError, 'error')
```
Comment (m-kuhn) [inline src/qml/qgismobileapp.qml:212]:
```suggestion
        displayToast(qsTr('Positioning error: %1').arg(positionSource.device.lastError), 'error')
```

Ltr langs etc

Comment (nirvn) [reply src/qml/qgismobileapp.qml]: @m-kuhn , fair point.

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**